doc/sphinx/Pacemaker_Development/helpers.rst

```diff
@@ -238,6 +224,17 @@ is no ``lib/common/tests/acls`` directory.
 
      check_PROGRAMS = pcmk_acl_required_test
 
+* Double check that ``-I$(top_srcdir)/lib/common`` is present in ``AM_CPPFLAGS``.
+
+* Double check the settings of variables at the top of ``Makefile.am``.  The
+  following block should be present:
+
+  .. code-block:: none
+
+     LDADD = $(top_builddir)/lib/common/libcrmcommon_test.la
```

Comment:
`-lcmocka`?

Reply:
Yes, added a patch to #2793 to fix this.